modules/security/security_manager.lua

```diff
@@ -0,0 +1,253 @@
+--- Module which provide SecurityManager type
+--
+-- *Dependencies:* `luaopenssl`, `security.security_constants`
+--
+-- *Globals:* none
+-- @module security.security_constants
+-- @copyright [Ford Motor Company](https://smartdevicelink.com/partners/ford/) and [SmartDeviceLink Consortium](https://smartdevicelink.com/consortium/)
+-- @license <https://github.com/smartdevicelink/sdl_core/blob/master/LICENSE>
+
+local openssl = require('luaopenssl')
+local securityConstants = require('security/security_constants')
+
+local SecurityManager = {}
+
+local function updateSecurityOfSession(security)
```

Comment:
@aderiabin Please add description for this and for other functions

Reply:
@dboltovskyi Done (see ab761db127418d0f025c8e5768470f626587e41c)